### ISSUE TITLE
As an Administrator, I want all content to be hidden to non-Administrative users until I choose to publish it

### ISSUE THREAD

**Issue by robyurkowski** [grounded/afterburnercms#27]
But I want this to be optional. If my site is set up to do automatic publishing, then content should go live as I create it. Otherwise, I should have to click a button to publish the content, and click a button to unpublish it.

This means that I should be able to stage content, so-to-speak.

What would be even _cooler_ but unnecessary at the moment would be the ability to batch publish groups of content, so I could prepare releases and then schedule them to go at a certain date. This should probably be another feature that is discussed, though.

This is a killer feature for my business, SuperEnterpriseFortune500PleasePickMe.

**Comment by parndt** [comment]
This is the same as unpublished revisions?

**Comment by knewter** [comment]
So yeah, revisions, with a published_at timestamp?

**Comment by robyurkowski** [comment]
Yep. But there's an additional catch in there that you can manually publish, as well, instead of simply publish-by-date.

**Comment by knewter** [comment]
Manually publish:

``` ruby
content.published_at = Time.zone.now
```

I JUST BLEW YOUR MIND

**Comment by robyurkowski** [comment]
MIND BLOWN
